tools/esdlcmd/esdl2ecl.cpp

```diff
@@ -269,6 +269,8 @@ class Esdl2EclCmd : public EsdlConvertCmd
             return EsdlCmdOptionMatch;
```

Comment:
was the change to libcouchbase above intentional?

Reply:
ugh. unintentional. I'll take a look.

Reply:
removed